README.md

```diff
@@ -14,30 +14,55 @@ Apollo. You may find references to these previous names in code or
 documentation.
 
 ## Community
-You can reach out to us through [Slack](https://cloud-native.slack.com/archives/C01TDE3GK0E) (#stackrox).  
-For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackrox.io/).
+You can chat directly with us on the [#stackrox channel in the CNCF Slack](https://www.stackrox.io/slack/).
+
+For event updates, blogs and other resources follow the StackRox community site at [stackrox.io](https://www.stackrox.io/).
 
 ## Table of contents
 
-* [Quick Installation via Helm](#quick-installation-via-helm)
-    + [First use](#first-use)
-* [Manual Deployment](#manual-deployment)
-* [Development](#development)
-    + [Quickstart](#quickstart)
+- [StackRox Kubernetes Security Platform](#stackrox-kubernetes-security-platform)
+  - [Community](#community)
+  - [Table of contents](#table-of-contents)
```

Comment:
Should we include the ToC and the Community if they are (technically) above the ToC?
If this ToC is generated automatically, this is totally fine, but it surprised me to find these entries here, that's why I'm asking

Reply:
moved ToC to the top. Added code of conduct and disclosure links